cpp/src/bitmask/null_mask.cu

```diff
@@ -258,14 +341,14 @@ rmm::device_buffer copy_bitmask(bitmask_type const *mask, size_type begin_bit,
     cudf::util::cuda::grid_config_1d config(number_of_mask_words, 256);
     copy_offset_bitmask<<<config.num_blocks, config.num_threads_per_block, 0,
                           stream>>>(
-        static_cast<bitmask_type *>(dest_mask.data()), mask, begin_bit,
+        static_cast<bitmask_type *>(dest_mask.data()), mask, begin_bit, end_bit,
```

Comment:
What's the point of passing in both the begin/end bit and the number of works? Aren't one of those redundant?

Reply:
Sure, I can remove number of words. end_bit is necessary for boundary checks.